strongbox-parent/pom.xml

```diff
@@ -577,7 +577,12 @@
                 <artifactId>spring-webmvc</artifactId>
                 <version>${version.spring}</version>
             </dependency>
-
+            <dependency>
```

Comment:
Just for my better understanding -- what are we using this for? I understand it's for XML, but I don't quite see where we're using it...